src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,32 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
```

Comment:
The prop name `activeTabId` should be changed to `activeId` to match the task requirements. Ensure consistency across all components.